v3/setup/klam-ssh.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+AZ=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
+REGION=${AZ::-1}
+ROLE_NAME="$(etcdctl get /klam-ssh/ROLE_NAME)"
+ENCRYPTION_ID="$(etcdctl get /klam-ssh/ENCRYPTION_ID)"
+ENCRYPTION_KEY="$(etcdctl get /klam-ssh/ENCRYPTION_KEY)"
+KEY_LOCATION_PREFIX="$(etcdctl get /klam-ssh/KEY_LOCATION_PREFIX)"
+IMAGE="$(etcdctl get /images/klam-ssh)"
+
+
+if [[ $REGION == "eu-west-1" ]]; then
+    KEY_LOCATION="-ew1"
+elif [[ $REGION == "ap-northeast-1" ]]; then
```

Comment:
should we make this list accessible to the widest swath of regions, or is unnecessary?